src/main/java/com/redhat/darcy/salesforce/RelatedList.java

```diff
@@ -20,12 +20,19 @@
 package com.redhat.darcy.salesforce;
 
 import static com.redhat.darcy.ui.Elements.element;
+import static com.redhat.darcy.ui.Elements.link;
 import static com.redhat.darcy.ui.Elements.text;
+import static com.redhat.synq.Synq.after;
+
+import java.time.temporal.ChronoUnit;
+
+import org.hamcrest.Matchers;
```

Comment:
I'm a little bit of a stickler about import order, this is the style for darcy projects:

import static *
(line break)
import com.redhat.darcy.*
(line break)
import * // everything except java imports (in other words all 3rd party imports)
(line break)
import java.*